src/saving/SaveHelper.cs

```diff
@@ -131,11 +129,48 @@ private static Save CreateSaveObject(MainGameState gameState, SaveInformation.Sa
         };
     }
 
+    private static string BuildQuicksaveFilename(uint quicksaveId)
+        => $"quick_save.{quicksaveId}.{Constants.SAVE_EXTENSION}";
```

Comment:
Our style guide doesn't say anything about this, but I'm kinda leaning towards requiring methods to use curly braces and not the lambda syntax.